src/compiler/checker.ts

```diff
@@ -2772,10 +2772,12 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
         return combined;
     }
 
-    function mergeSymbolTable(target: SymbolTable, source: SymbolTable, unidirectional = false) {
+    function mergeSymbolTable(target: SymbolTable, source: SymbolTable, unidirectional = false, parent?: Symbol) {
         source.forEach((sourceSymbol, id) => {
             const targetSymbol = target.get(id);
-            target.set(id, targetSymbol ? mergeSymbol(targetSymbol, sourceSymbol, unidirectional) : getMergedSymbol(sourceSymbol));
+            const merged = targetSymbol ? mergeSymbol(targetSymbol, sourceSymbol, unidirectional) : getMergedSymbol(sourceSymbol);
+            merged.parent ??= parent;

```

Comment:
from discussion with @iisaduan -- do we know which merged symbols don't have parents? Is it related to whether `sourceSymbol` (or target) has a parent? Isabel tracked it down to exports lists=no parent; exported interface=parent. Is there something missing in the binder when parents are set?